ERROR:__main__:2022-12-10 18:54:16,242:local variable 'portfolio_dict' referenced before assignment
Traceback (most recent call last):
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\main_strategy_run.py", line 128, in <module>
    strat_obj_dict[j].update()
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\strat7.py", line 609, in update
    self._print_details("updating")
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\strat7.py", line 202, in _print_details
    print(self.mybet.get_event_holdings(self.event_id))
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\mybets.py", line 44, in get_event_holdings
    port_dict = self._get_portfolio_dict()
  File "D:\Synced folder 2\tradex_opinionTrading\codes_2\mybets.py", line 41, in _get_portfolio_dict
    return portfolio_dict
UnboundLocalError: local variable 'portfolio_dict' referenced before assignment
